app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,30 @@
-<h2>Log in</h2>
+  <div class="container content">
+  <h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+  <span><i class="fa fa-film" style="font-size:48px; color:white;"></i></span>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    
+    <label class="label control-label">E-mail</label>
```

Comment:
Can't you use dafault bootstrap form labels?
Like this:
```erb
<div class="form-group">
    <%= f.label :email %>
    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "E-mail" %>
</div>
```

Reply:
And you are missing `form-control` call on the input fields
`<%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "E-mail" , class: "form-control"%>`

Take a look 
https://getbootstrap.com/docs/4.0/components/forms/